contrib/cmd/fs_idmap/fs_idmap.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"golang.org/x/sys/unix"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		log.Fatalf("usage: %s path_to_mount_set_attr", os.Args[0])
+	}
+
+	src := os.Args[1]
+	treeFD, err := unix.OpenTree(-int(unix.EBADF), src, uint(unix.OPEN_TREE_CLONE|unix.OPEN_TREE_CLOEXEC|unix.AT_EMPTY_PATH|unix.AT_RECURSIVE))
```

Comment:
Same weird use of errno code. You can use `-1` here I think.

Reply:
This suggestion slipped through the cracks, fixed now!